linux_os/guide/system/accounts/accounts-restrictions/root_logins/use_pam_wheel_for_su/bash/shared.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
+
+if ! grep -q pam_wheel /etc/pam.d/su; then
```

Comment:
This does not do what you expect - I tried this condition and it got evaluated as true when there was no `pam_wheel` and evaluated as false when `pam_wheel` was present in `/etc/pam.d/su`.